pkg/controller/nodelink/nodelink_controller_test.go

```diff
@@ -705,3 +706,67 @@ func TestIndexMachineByInternalIP(t *testing.T) {
 		}
 	}
 }
+
+func TestUpdateNodeRef(t *testing.T) {
+	testCases := []struct {
+		machine            *mapiv1beta1.Machine
+		node               *corev1.Node
+		nodeRef            *corev1.ObjectReference
+		nodeReadinessCache map[string]bool
```

Comment:
where is this being used?

Reply:
https://github.com/openshift/machine-api-operator/pull/336/files#diff-69885f1e6e1f6b4cb889ce80021c95a1R273